examples/gzip/docker-compose.yaml

```diff
@@ -0,0 +1,24 @@
+version: "3.7"
+services:
+  envoy-stats:
+    build:
+      context: .
+      dockerfile: Dockerfile-gzip
+    volumes:
+      - ./gzip-envoy.yaml:/etc/gzip-envoy.yaml
+    networks:
+      - envoymesh
+    ports:
+      - "8001:8001"
+      - "8002:8002"
```

Comment:
i dont think all of these ports need to be opened - only the ones that are queried by the host

Reply:
8001 and 8002 are both queried by host.

Reply:
ok - i see